libc/cmake/modules/LLVMLibCTestRules.cmake

```diff
@@ -1,11 +1,6 @@
 function(_get_common_test_compile_options output_var c_test flags)
   _get_compile_options_from_flags(compile_flags ${flags})
 
-  # Remove -fno-math-errno if it was added.
-  if(LIBC_ADD_FNO_MATH_ERRNO)
-    list(REMOVE_ITEM compile_options "-fno-math-errno")
-  endif()
```

Comment:
Did you intend to remove this?

Reply:
IIRFC yes, Oh I just realized that's _removing_ `-fno-math-error`

let me revert that change